data/s3dis/indoor3d_util.py

```diff
@@ -0,0 +1,74 @@
+import glob
+import numpy as np
+import os
+
+# -----------------------------------------------------------------------------
+# CONSTANTS
+# -----------------------------------------------------------------------------
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+g_classes = [
```

Comment:
What does g_ here mean? The variable name is a little strange.

Reply:
I guess g means global because they are some global variables? Anyway I will modify them into names that are easier to understand.